src/js/inject.js

```diff
@@ -465,6 +507,40 @@ $('body').on('click', '#open-modal', (ev) => {
   }
 });
 
+const findBiggestBitrate = (videos) => {
+  return videos.reduce((max, x) => {
+    return (x.bitrate || -1) > (max.bitrate || -1) ? x : max;
+  });
+};
+
+const getMediaForElement = (element) => {
+  const chirp = getChirpFromElement(element);
+  const urls = [];
+
+  chirp.entities.media.forEach((item) => {
+    switch (item.type) {
+      case 'video':
+      case 'animated_gif':
+        urls.push(findBiggestBitrate(item.video_info.variants).url);
+        break;
+      case 'photo':
+        urls.push(item.media_url_https);
+        break;
+      default:
+        throw new Error(`unsupported media type: ${item.type}`);
+    }
+  });
+
+  return urls;
+};
+
+// eslint-disable-next-line
```

Comment:
I see, I guess you could use the [events API](https://github.com/zenorocha/clipboard.js/#events) in that case. But if you don't want to bother that's fine by me. Maybe just add a comment in the so whoever sees that line understands why we do this.